entity-service/src/main/kotlin/com/egm/stellio/entity/config/WebConfig.kt

```diff
@@ -1,15 +1,26 @@
 package com.egm.stellio.entity.config
 
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
 import org.springframework.http.codec.ServerCodecConfigurer
+import org.springframework.web.reactive.config.CorsRegistry
 import org.springframework.web.reactive.config.EnableWebFlux
 import org.springframework.web.reactive.config.WebFluxConfigurer
 
+@Profile("!test")
 @Configuration
 @EnableWebFlux
 class WebConfig : WebFluxConfigurer {
 
+    @Value("\${allowed.origins}")
+    val allowedOrigins: List<String>? = null
```

Comment:
this is a bit vague when read out of context.

what about `application.cors.allowed_origins`?

Reply:
I'll update it